pull-through-cache/README.md

```diff
@@ -0,0 +1,40 @@
+# Pull-Through Cache
+
+This is an example of how you might configure and use a pull-through cache to augment your Earthly CI builds. There are a couple parts within this folder, some of which are meant for illustration only. If you need additional help configuring a pull-through cache with 
```

Comment:
Sentence is unfinished...

```
If you need additional help configuring a pull-through cache with ...
```

Reply:
I would add a bit more color about where the situations in which this pull-through cache may be useful. For example, it's not as useful for individual use (even though the Earthfile changes the user's Earthly config), but it's most relevant for CI, to avoid hitting vendor rate limiting.

Reply:
As I was writing this example I came across a surprising number of people who run one locally! But yeah I'll add a list